demo/src/App-ReactState.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
```

Comment:
Would you mind renaming `demo` to `demo-react` to make them peers?  With that, the root npm script of `demo` will need to be updated to be `demo-react` as well.